android/src/com/google/zxing/client/android/camera/CameraConfigurationManager.java

```diff
@@ -183,6 +183,10 @@ void setDesiredCameraParameters(OpenCamera camera, boolean safeMode) {
 
     parameters.setPreviewSize(bestPreviewSize.x, bestPreviewSize.y);
 
+    //SetRecordingHint to true also a workaround for low framerate on Nexus 4
```

Comment:
Ah, I think this should go in the 'safe mode' block above. That way if there's some device for which this causes an exception, at least it can try not setting it. I don't think we need to add an option to disable at this point.